src/logic/asset-converter.ts

```diff
@@ -0,0 +1,83 @@
+import ICommandTrigger from '../ports/ICommandTrigger'
+import { CrunchConversionDefaultOptions, CrunchConversionOptions } from '../types/crunch-conversion'
+import ProcessWrapper from '../types/processWrapper'
+
+export class Conversion {
```

Comment:
I think we can reduce this whole class to only one function call. We tend to avoid classes to make testing easier